packages/paste-core/components/chat-composer/src/ChatComposer.tsx

```diff
@@ -82,6 +108,13 @@ export interface ChatComposerProps extends Omit<ContentEditableProps, "style" |
    * @memberof ChatComposerProps
    */
   onChange?: OnChangeFunction;
+  /**
+   *
+   * @default null
+   * @type {React.MutableRefObject<LexicalEditor | null>}
+   * @memberof ChatComposerProps
+   */
+  editorInstanceRef?: React.MutableRefObject<LexicalEditor | null>;
```

Comment:
suggestion(non-blocking): what about `editorRef` as the name of the prop so it's the same as what we're suggesting consumers name the value, and a little shorter?

Reply:
Totally happy to do that. I named it this intentionally so our API highlights it accesses the Lexical instance within the nested context. I guess with the type being LexicalEditor it should be explicit enough right?

Reply:
I could go either way, definitely see your reasoning. Maybe in the story you can call both editorInstanceRef so that we're passing the same named value to the prop. Super tiny nit just makes things a little cleaner. (so it would be a change to the story, not to the API)